gocd/alp.target.gocd.yaml

```diff
@@ -0,0 +1,225 @@
+        resources:
+        - staging-bot
+        tasks:
+        - script: |-
+            sle-prjmgr-tools --jira-instance $JIRA_INSTANCE --osc-config $OSC_CONFIG ibs_to_jira --jira-pat $JIRA_PAT -p SUSE:ALP:Products:Installer:1.0
+
+    - Release.Images.To.Publish:
+        approval: manual
+        roles:
+        - SLE
+        environment_variables:
+          OSC_CONFIG: /home/go/config/oscrc-totest-manager
+        resources:
+        - staging-bot
+        tasks:
+        - script: |-
+            osc -A https://api.suse.de release SUSE:ALP:Products:Installer:1.0:ToTest
+            sleep 600
+            while (osc -A https://api.suse.de/ api "/build/SUSE:ALP:Products:Installer:1.0:PUBLISH/_result?view=summary&repository=images" | grep "result project" | grep -v 'code="published" state="published">'); do
+                echo PENDING
+                sleep 600
+            done
+            osc -A https://api.suse.de/ api "/build/SUSE:ALP:Products:Installer:1.0:PUBLISH/_result?view=summary&repository=images" | grep "result project" | grep 'code="published" state="published">' && echo PUBLISHED
+
+  Installer.ImagesManual:
+    group: ALP.Target
+    lock_behavior: unlockWhenFinished
+    materials:
+      repos:
+        git: git://botmaster.suse.de/suse-repos.git
+        auto_update: true
+        whitelist:
+          - SUSE:ALP:Products:Installer:1.0_-_images.yaml
+        destination: repos
+      scripts:
+        auto_update: true
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+        whitelist:
+          - DO_NOT_TRIGGER
+        destination: scripts
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-staging-bot
+    stages:
+    - Relase.Images.To.Test:
+        approval: manual
+        roles:
+          - SLE
+        jobs:
+          Check.Images.Finish:
+            resources:
+              - staging-bot
+            tasks:
+              - script: |
+                  export PYTHONPATH=scripts
+                  while (! ./scripts/gocd/verify-repo-built-successful.py -A https://api.suse.de -p SUSE:ALP:Products:Installer:1.0 -r images); do
+                    sleep 300
+                  done
+          Release.To.Test:
+            environment_variables:
+              OSC_CONFIG: /home/go/config/oscrc-totest-manager
+            resources:
+              - staging-bot
+            tasks:
+              - script: osc -A https://api.suse.de release SUSE:ALP:Products:Installer:1.0
+
+    - Release.Images.To.Publish:
+        approval: manual
+        roles:
+          - SLE
+        environment_variables:
+          OSC_CONFIG: /home/go/config/oscrc-totest-manager
+        resources:
+          - staging-bot
+        tasks:
+          - script: osc -A https://api.suse.de release SUSE:ALP:Products:Installer:1.0:ToTest
+
+  Installer.ReleaseScc:
+    group: ALP.Target
+    parameters:
+      product: installer-1.0
+      target: MyTarget
```

Comment:
Is this correct? Looks like a placeholder

Reply:
it looks correct, other alp pipelines have the similar params feed to scc